docs/mysql.md

```diff
@@ -1,3 +1,75 @@
+# `mysql` Metrics Receiver
+
+The mysql receiver can retrieve stats from your mysql instance by connecting as a monitoring user.
+
+
+## Prerequisites
+
+It is recommended that you create a dedicated monitoring user. The monitoring user must be granted the following permissions:
+
+```
+PROCESS ON *.*
+SELECT ON INFORMATION_SCHEMA.INNODB_METRICS
+```
+
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your mysql configuration.
+
+To configure a receiver for your mysql metrics, specify the following fields:
+
+| Field                 | Default                   | Description |
+| ---                   | ---                       | ---         |
+| `type`                | required                  | Must be `mysql`. |
+| `endpoint`            | `localhost:3306`          | The hostname:port or socket path used by mysql |
+| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+| `username`            |                           | The username used to connect to the server. |
+| `password`            |                           | The password used to connect to the server. |
+| `transport`           |                           | Defines the network to use for connecting to the server. |
+
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    mysql_metrics:
+      type: mysql
```

Comment:
This example configuration implies that the user needs to set all of these fields; can we have two examples, the first with none of the fields set, and the second with a description like "TCP connection with a username and password"?

Reply:
added

Reply:
Resolving this thread as it has been addressed in https://github.com/GoogleCloudPlatform/ops-agent/commit/c55e07c2a4a26fe674f3feb49536b32e7521642f.